### PR TITLE
Including neccessary change to make bookstack work

### PR DESCRIPTION
**Pull request by thunfischbrot**

### CODE REVIEW

File: post_install.sh [hunk 1]
```diff
@@ -18,10 +18,54 @@ sed -i '' -e 's/;listen.group = www/listen.group = www/g' /usr/local/etc/php-fpm
...
+        }
+
+        error_page   500 502 503 504  /50x.html;
+            location = /50x.html {
+            root   /usr/local/www/nginx-dist;
+        }
+    }
+}
+EOL
+
```
Comment (Adirael) [inline post_install.sh]: I think this part is not needed, the iocage creation will copy files from the overlay folder into the proper location (see https://github.com/Adirael/iocage-plugin-bookstack/blob/master/overlay/usr/local/etc/nginx/nginx.conf)

**Review by Adirael — CHANGES_REQUESTED** [review]
Thanks for the PR! Good timing, I need to reinstall this since I broke my FreeNAS installation a few days ago.

File: post_install.sh (hunk 1 above)
Comment (thunfischbrot) [reply post_install.sh]: Yes, good point. Overlooked that, I am pretty new to iocages. I made additional fixes, such as adapting the packages needed in the iocage description.

**Review by Adirael — APPROVED** [review]
Looks great! Thanks a lot for fixing this up :)

**Comment by thunfischbrot** [comment]
Sure, thanks for the template!

**Pull request merged**